src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message should be more informative and indicate the specific type of machine that is working. Consider specifying that an 'Excavator' is working.